.buildkite/pipeline.testing.yml

```diff
@@ -1,28 +0,0 @@
----
```

Comment:
Due to the nature of the pipelines, we should leave this file in place... you can have a dummy step that just runs `echo "Do some real tests later"`.